tasks/main.yml

```diff
@@ -1,42 +1,50 @@
 ---
 - name: Include OS-specific variables.
   include_vars: "{{ ansible_os_family }}.yml"
+  tags: [ ntp ]
```

Comment:
Please remove the addition of the tag per task (I prefer to not add tags to every task in my roles, rather if I need a tag for something like NTP, I'll add it in the playbook when calling the role so it applies to the entire role).